mkdocs.yml

```diff
@@ -1,62 +1,25 @@
-# yaml-language-server: $schema=https://squidfunk.github.io/mkdocs-material/schema.json
-
-site_name: ""
+site_name: "" # already part of the logo
+site_description: "Ask-a-Question Documentation"
 site_url: https://idinsight.github.io/aaq-core/
-extra_css:
-  - stylesheets/extra.css
-
+repo_url: https://github.com/IDinsight/aaq-core
+copyright: "© 2024 IDinsight"
 extra:
   generator: false
   social:
     - icon: material/shield-lock
       link: https://www.idinsight.org/about/privacy-policy/
       name: Privacy Policy
 
-copyright: "© 2024 IDinsight"
-
-theme:
-  name: material
-  logo: images/logo-light.png
-  favicon: images/favicon.png
-  font:
-    text: "Inter"
-    code: "Roboto Mono"
-  palette:
-    # Palette toggle for dark mode
-    - scheme: slate
-      primary: custom
-      toggle:
-        icon: material/brightness-4
-        name: Switch to light mode
-    # Palette toggle for light mode
-    - scheme: default
-      primary: custom
-      toggle:
-        icon: material/brightness-7
-        name: Switch to dark mode
-
-  features:
-    - navigation.tabs
-    - navigation.tabs.sticky
-    - content.code.annotate
-    - navigation.sections
-    - navigation.indexes
-    - navigation.top
-  icon:
-    admonition:
-      note: material/alert-circle
-    annotation: material/help-circle
-
-repo_url: https://github.com/IDinsight/aaq-core
-
 nav:
   - Home:
       - "": index.md
       - Usage: usage.md
       - Roadmap: roadmap.md
+
   - Latest Updates:
       - blog/index.md
-  - Core Components:
+
+  - Components:
```

Comment:
Thoughts on pulling Integrations out as toplevel menu?

Also, is there a way to create sections on the left menu to distinguish User-facing vs. Internal components?

<img width="1063" alt="image" src="https://github.com/IDinsight/aaq-core/assets/7042047/c252f71a-b922-48f8-972c-2fb247f3b253">

Reply:
agreed